src/ForwardRomanPot_geo.cpp

```diff
@@ -43,7 +43,8 @@ static Ref_t create_detector(Detector& description, xml_h e, SensitiveDetector s
 
     Box    m_solid(mod_width / 2.0, mod_height / 2.0, mod_total_thickness / 2.0);
     Volume m_volume(m_nam, m_solid, vacuum);
-    m_volume.setVisAttributes(description.visAttributes(x_mod.visStr()));
+    //m_volume.setVisAttributes(description.visAttributes(x_mod.visStr()));
```

Comment:
Remove.

Reply:
I added a comment to keep this line - I commented it out to try forcing a single color since the RP visualization is not working in the GeoViewer.